source/_docs/build-https.md

```diff
@@ -1,17 +0,0 @@
----
-title: Developing with HTTPS
```

Comment:
I think we need to preserve this content somewhere, but maybe we can deal with that in http-to-https